### ISSUE TITLE
Weather tab during non forecasting season is confusing

### ISSUE THREAD

**Issue by yuliadub** [stevekuznetsov/avalanche-forecast#280]
When weather forecasting season ends, the message displayed on the weather tab makes it seem like something is wrong (also there is a button that you can click that doesnt go anywhere) The error message also blocks weather stations for the zone. 

I think it would be great to:
- separate weather forecast and weather station failure paths, so one not loading doesnt block the other
- add a condition that is different for error loading when the timing is outside forecasting season (similar to how forecast become grey? may need to consult with design on this) 

What I see today:
![IMG_7847](https://github.com/stevekuznetsov/avalanche-forecast/assets/9418602/aa7e3b45-ed80-4b96-8e7e-0a9a50d9afdc)

**Comment by stevekuznetsov** [comment]
Yeah, we likely want different error codes at the HTTP layer as well - and need to handle this for the NWAC weather as well as the NAC one.

**Comment by stevekuznetsov** [comment]
@yuliadub this is #285 - so I think we are disambiguating between 404 and 500, but just not the way we want :)
